spec/e2e/nodesets/el7.yml

```diff
@@ -9,7 +9,9 @@ HOSTS:
     docker_cmd:
       - '/usr/sbin/init'
     docker_image_commands:
-      - 'yum install -y yum-utils wget which cronie iproute initscripts'
+      - 'yum install -y yum-utils wget which cronie iproute initscripts strace'
+      - wget --no-check-certificate https://copr.fedorainfracloud.org/coprs/jsynacek/systemd-backports-for-centos-7/repo/epel-7/jsynacek-systemd-backports-for-centos-7-epel-7.repo -O /etc/yum.repos.d/jsynacek-systemd-centos-7.repo
+      - yum update -y systemd
```

Comment:
A little hacky - but I found the centos:7 issue boiled down to systemd not booting correctly due to cgroup and the inability to mount this directory.

```
[jeff src(fix-shellcheck)] 🐯  podman run --rm -t --name test 6c329ea49 /sbin/init 
Failed to mount cgroup at /sys/fs/cgroup/systemd: No such file or directory
[!!!!!!] Failed to mount API filesystems, freezing.
```